### PR TITLE
Move sidebar to top of course about in mobile

### PR DESCRIPTION
**Pull request by caesar2164**

### CODE REVIEW

**Comment by caesar2164** [comment]
Need to add window resize handler.

File: lagunita/lms/templates/courseware/course_about_js_extra.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    var mobileWidth = 768;
+
+    $(document).ready(function() {
+        var contentWidth = $('#content').width();
+        if (contentWidth < mobileWidth) {
+            var courseDetails = $('#content .details');
+            var courseSidebar = $('#content .course-sidebar');
+            var courseSidebarHeight = courseSidebar.height();
+
+            courseDetails.css("margin-top", courseSidebarHeight);
```
Comment (stvstnfrd) [inline lagunita/lms/templates/courseware/course_about_js_extra.html:11]: Single quotes for non-interpolated strings.

File: lagunita/lms/templates/courseware/course_about_js_extra.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<script type="text/javascript">
+    var mobileWidth = 768;
```
Comment (stvstnfrd) [inline lagunita/lms/templates/courseware/course_about_js_extra.html:2]: Investigate options for sticking this in a variable. Ideally, the same variable will be used by the sass.

**Review by stvstnfrd — CHANGES_REQUESTED** [review]

File: lagunita/lms/templates/courseware/course_about_js_extra.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+    $(document).ready(function() {
+        var contentWidth = $('#content').width();
+        if (contentWidth < mobileWidth) {
+            var courseDetails = $('#content .details');
+            var courseSidebar = $('#content .course-sidebar');
```
Comment (stvstnfrd) [inline lagunita/lms/templates/courseware/course_about_js_extra.html:8]: Move this to the top of the function then bail if length != 1.

```js
var courseSidebar = $('#content .course-sidebar');
if (courseSidebar.length !== 1) {
    return;
}
// continue w/ resizing/etc...
```

**Comment by caesar2164** [comment]
this is getting pushed to after the Ginkgo merge

**Pull request closed without merging**